corehq/apps/domain/views.py

```diff
@@ -1085,6 +1087,16 @@ def dispatch(self, request, *args, **kwargs):
 
     def post(self, request, *args, **kwargs):
         emails = request.POST.get('emails', []).split()
+        invalid_emails = []
+        for email in emails:
+            try:
+                validate_email(email)
+            except ValidationError:
+                invalid_emails.append(email)
+        if invalid_emails:
+            message = ('The following e-mail addresses failed validation: ' +
```

Comment:
I'm not sure if it would be clear to non technical users that "failed validation" means "has invalid characters" in this case (it could also be interpreted as meaning "this email address is already in use", or some other reason for not being accepted). Maybe something along the lines of "The following e-mail addresses have invalid characters: " or something like that? (Not sure if it also checks for the existence "@" and stuff like that as well...)